### PR TITLE
Standardise metric naming

### PR DESCRIPTION
**Pull request by Dalamar42**
- Standardise metric naming across Kafka workers and in
  ContentHashGenerator for consistency and to make it easier to
  create standardised views for them in Grafana

### CODE REVIEW

File: atlas-processing/src/main/java/org/atlasapi/system/bootstrap/workers/BootstrapWorkersModule.java [hunk 1]
```diff
@@ -277,17 +306,21 @@ KafkaConsumer topicReadWriter() {
...
+                metricsModule.metrics()
         );
+
         return bootstrapQueueFactory()
                 .createConsumer(
                         worker,
                         new EntityUpdatedLegacyMessageSerializer(),
                         eventChanges,
-                        "SeparatingEventBootstrap"
+                        workerName
```
Comment (AdamDM) [inline atlas-processing/src/main/java/org/atlasapi/system/bootstrap/workers/BootstrapWorkersModule.java]: duplicate code? Extract the return block and pass what changes in

File: atlas-processing/src/main/java/org/atlasapi/messaging/ContentEquivalenceUpdatingWorker.java [hunk 2]
```diff
@@ -15,6 +12,7 @@
 import com.codahale.metrics.Meter;
 import com.codahale.metrics.MetricRegistry;
 import com.codahale.metrics.Timer;
+import com.google.api.client.repackaged.com.google.common.base.Throwables;
```
Comment (emilssolmanis) [inline atlas-processing/src/main/java/org/atlasapi/messaging/ContentEquivalenceUpdatingWorker.java]: You're wrong.

Comment (Dalamar42) [inline atlas-processing/src/main/java/org/atlasapi/messaging/ContentEquivalenceUpdatingWorker.java]: Err, yes... Fixing

File: atlas-processing/src/main/java/org/atlasapi/system/bootstrap/workers/BootstrapWorkersModule.java (hunk 1 above)
Comment (Dalamar42) [inline atlas-processing/src/main/java/org/atlasapi/system/bootstrap/workers/BootstrapWorkersModule.java]: https://programmers.stackexchange.com/questions/300043/illusory-code-duplication

We already have the builder to standardise creating consumers. Adding a method to wrap that builder would be overengineering a bit in this case.

**Pull request merged**